docs/docker-images/buildkit-standalone.md

```diff
@@ -0,0 +1,61 @@
+This image contains `buildkit` with some Earthly-specific setup. This is what Earthly will start when using a local daemon. You can also start it up yourself and use it as a remote/shared buildkit daemon.
+
+## Quickstart
```

Comment:
Docker Hub images often have some information about the tags available. We should mention that `latest` points to the latest released version and that `prerelease` points to the latest build on `main`.

For `buildkit`, we've only tested versions of the image that have the exact same version as `earthly` itself. Perhaps this is something we can improve in the future, but for now perhaps we should recommend that people use the exact same version of earthly and buildkit.